src/makeCalculator.js

```diff
@@ -37,7 +37,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  return {
+    result: 0,
+    add: function(number) {
+      this.result += number;
+
+      return this;
```

Comment:
There is no need to return `this` for each method. Only reset and operate methods should be called in chain